.buildkite/cache-builder.yml

```diff
@@ -13,18 +13,22 @@ common_params:
         repo: "woocommerce/woocommerce-ios/"
   # Common environment values to use with the `env` key.
   - &common_env
-    IMAGE_ID: xcode-13.3.1
+    IMAGE_ID: xcode-13.4.1
 
 steps:
 
   #################
   # Build the CocoaPods Base Cache
-  # 
+  #
   # This prevents the base cache from infinite growth caused by storing every
   # version of every pod we've ever used.
   #################
   - label: ":cocoapods: Rebuild CocoaPods cache"
     command: |
+      # Workaround for https://github.com/Automattic/buildkite-ci/issues/79
+      echo "--- :rubygems: Fixing Ruby Setup"
+      gem install bundler
+
```

Comment:
Verified this works with a [manual build](https://buildkite.com/automattic/woocommerce-ios/builds/6077):

<img width="1205" alt="image" src="https://user-images.githubusercontent.com/1218433/173713084-ecea2cb5-a5ae-47de-a6a1-2d011948f6eb.png">